modules/standard/IO.chpl

```diff
@@ -4563,8 +4563,8 @@ proc read(type t ...?numTypes) throws {
   return stdin.read((...t));
 }
 /* Equivalent to ``stdin.readline``.  See :proc:`channel.readline` */
-proc readline(arg: [] uint(8), out numRead : int, start = arg.domain.low,
-              amount = arg.domain.high - start + 1) : bool throws
+proc readline(arg: [] uint(8), out numRead : int, start = arg.domain.alignedLow,
+              amount = arg.domain.size) : bool throws
```

Comment:
I think this should match the other readline. As-is, this might result in a change in behavior if ``start > arg.domain.alignedLow``, in which case the method could return false.

Reply:
Made strided cases a compiler error here, as with the channel method version in f0bb9f38aae7aeb071b3d1b8dd0ca775bb38c88b